scripts/o3de/tests/ExportScripts/test_export_source_android.py

```diff
@@ -0,0 +1,880 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+import logging
+import pytest
+import pathlib
+from unittest.mock import patch, create_autospec, MagicMock, Mock, PropertyMock
+from o3de.export_project import O3DEScriptExportContext
+from o3de.command_utils import O3DEConfig
+
+# Because the export scripts are standalone, we have to manually import them to test
+import o3de.utils as utils
+from o3de import android_support
+import platform
+utils.prepend_to_system_path(pathlib.Path(__file__).parent.parent.parent / 'ExportScripts')
+import export_source_android as expa
+
+
+
+#helper to generate settings file
+def create_dummy_commands_settings_file(build_config='profile', tool_config='profile', archive_format='none',
```

Comment:
if this is getting defined in multiple tests we should move it to a test utility .py file